packages/tooltip/src/Tooltip.js

```diff
@@ -58,13 +58,16 @@ Tooltip.propTypes = {
   isVisible: PropTypes.bool,
   align: PropTypes.oneOf(values(alignments)),
   zIndex: PropTypes.number,
-  bondariesSelector: PropTypes.string
+  bondariesSelector: PropTypes.string,
+  showArrow: PropTypes.bool,
+  padding: PropTypes.string
 };
 
 Tooltip.defaultProps = {
   isVisible: true,
   align: alignments.LEFT,
-  zIndex: 1000
+  zIndex: 1000,
+  showArrow: true
```

Comment:
if the default is to display the arrow, maybe the property could be a flag to hide it instead, `hideArrow` that if present hides it.